.ci/test_r_package.sh

```diff
@@ -4,7 +4,7 @@
 CRAN_MIRROR="https://cloud.r-project.org/"
 R_LIB_PATH=~/Rlib
 mkdir -p $R_LIB_PATH
-echo "R_LIBS=$R_LIB_PATH" > ${HOME}/.Renviron
+export R_LIBS=$R_LIB_PATH
```

Comment:
I made a mistake while developing...code with `Rscript --vanilla -e` failed to load a library because `--vanilla` ignores the `.Renviron`

I think we should remove this and use normal environment variables instead, to make the code a little more explicit and easier to reason about.